backend/test_observer/controllers/artefacts/models.py

```diff
@@ -93,3 +93,8 @@ class ArtefactBuildDTO(BaseModel):
 
 class ArtefactPatch(BaseModel):
     status: ArtefactStatus
+
+
+class RerunArtefactTestExecutionsRequest(BaseModel):
+    test_execution_status: TestExecutionStatus | None = None
+    test_execution_review_decision: list[TestExecutionReviewDecision] | None = None
```

Comment:
The `review_decision` is defined as a set in the `TestExecutionDTO`, although here it wouldn't really change a lot, if you want we can make it a set